Prepare for v2.1.0 (#92)
bottlerocket-os/bottlerocket-kernel-kit: CHANGELOG.md

# v2.1.0 (2024-04-02)

## OS Changes
 * Update kernel-6.1 from 6.1.130-139.222 to 6.1.131-143.221 ([#89])
 * Update kernel-5.15 from 5.15.178-120.187 to 5.15.179-121.185 ([#91])

## Build Changes

>  * Move NVIDIA helper binaries to `usr/bin` and provide symlinks to the previous location `/usr/libexec`

Reviewer: ```suggestion
 * Move NVIDIA helper binaries to `usr/bin` and provide symlinks to the previous location `/usr/libexec` ([#84])
```

Author: Thank you!

>  * Move NVIDIA helper binaries to `usr/bin` and provide symlinks to the previous location `/usr/libexec` ([#84])

Reviewer 2: ```suggestion
 * Move NVIDIA helper binaries to standard filesystem location ([#84])
```

Author: changed

# v2.1.0 (2024-04-02)

## OS Changes
 * Update kernel-6.1 from 6.1.130-139.222 to 6.1.131-143.221 ([#89])
 * Update kernel-5.15 from 5.15.178-120.187 to 5.15.179-121.185 ([#91])

[#89]: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/89

> [#91]: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/91

Reviewer: can you organize links at the bottom of this release's section like previous releases?

Author: changed

## Build Changes

>  * Move NVIDIA helper binsaries to `usr/bin` and provide symlinks at the previous location `/usr/libexec`

Reviewer: ```suggestion
 * Move NVIDIA helper binaries to `usr/bin` and provide symlinks to the previous location `/usr/libexec` ([#84])
```
